README.md

```diff
@@ -70,152 +67,41 @@
 - [License](#license)
 
 ## Quick start
-To get started quickly building applications, we recommend using the [`stack`](https://docs.haskellstack.org/en/stable/README/) or [`nix`](https://nixos.org/nix) package managers. Obtaining [`GHCJS`](https://github.com/ghcjs/ghcjs) is required as a prerequisite. `stack` and `nix` make this process easy, if you're using `cabal` we assume you have [obtained `GHCJS`](https://github.com/ghcjs/ghcjs#installation) by other means.
+To get started quickly building applications, we recommend using the [`nix`](https://nixos.org/nix) package manager with miso's binary cache provided by [`cachix`](https://haskell-miso.cachix.org/). It is possible to use [`stack`](https://docs.haskellstack.org/en/stable/README/) to build GHCJS projects, but support for procuring `GHCJS` has been removed [as of stack 2.0](https://github.com/commercialhaskell/stack/issues/4086).
+
+Obtaining [`GHCJS`](https://github.com/ghcjs/ghcjs) is required as a prerequisite. `nix` makes this process easy, if you're using `cabal` we assume you have [obtained `GHCJS`](https://github.com/ghcjs/ghcjs#installation) by other means.
```

Comment:
Saying that GHCJS is a prerequisite is confusing here in case of nix, if I didn't know better I would think it means that you still need to install GHCJS before even trying to build the project with nix. Again, I would move this explanation to "advanced" and just leave a simple "use nix like this to build this project including everything it needs automatically"